public/js/plantPlacement.js

```diff
@@ -1,229 +1,227 @@
+const article = document.querySelector('#circle_1');
 
 try {
   data = JSON.parse(article.dataset.plantedplants);
+} catch (e) {
 }
-catch(e) {
-}
-let len = data.length;
+const len = data.length;
 
 function defineUniquePlants() {
   $.each(data, (index, value) => {
-      if ($.inArray(value.plant, uniquePlants) === -1) {
-          uniquePlants.push(value.plant);
-      }
+    if ($.inArray(value.plant, uniquePlants) === -1) {
+      uniquePlants.push(value.plant);
+    }
   });
 }
 
-function updateSelector(selectorName,plantId) {
+function updateSelector(selectorName, plantId) {
   $(selectorName).val(plantId);
 }
 function checkPlants(x, y) {
-  for (var k = 0; k < len; k++) {
-    if (data[k] ) {
-        if (data[k].y == y && data[k].x == x) {
-          val = data[k].plant;
-          console.log(selectors);
-          a = uniquePlants.indexOf(data[k].plant.toString());
-          // uniquePlants[a]='used';
-          a += 1;
-          console.log('ddd',uniquePlants,a);
-          var sss= '#plant';
-          updateSelector(sss+a,data[k].plant);
+  for (let k = 0; k < len; k++) {
+    if (data[k]) {
+      if (data[k].y == y && data[k].x == x) {
+        val = data[k].plant;
+        console.log(selectors);
+        a = uniquePlants.indexOf(data[k].plant.toString());
+        // uniquePlants[a]='used';
+        a += 1;
+        console.log('ddd', uniquePlants, a);
+        const sss = '#plant';
+        updateSelector(sss + a, data[k].plant);
         return a;
-        }
       }
     }
-    return false;
+  }
+  return false;
 }
 
-  function drawMod(shape) {
-    let moduleColor = '#7FD674';
-    let article = document.querySelector('#circle_1');
-    //  let data = JSON.parse(article.dataset.plantedPlants)
-      console.log(article.dataset.plantedplants);
-    var draw = SVG().addTo('#circle_1').size(900, 300)
-    console.log(shape);
-    if (shape == 'Rtriangle') {
-      var triangle = draw.polygon('0,0 900,1300 900,0').fill(moduleColor).stroke({
-        width: 1
-      })
-    }
-    if (shape == 'Ltriangle') {
-      var triangle = draw.polygon('0,0 900,300 900,0').fill(moduleColor).stroke({
-        width: 1
-      })
-    }
-    if (shape == 'rectangle') {
-      let rect = draw.rect(900, 300).attr({
-        fill: moduleColor
-      });
-    }
+function drawMod(shape) {
+  const moduleColor = '#7FD674';
+  const article = document.querySelector('#circle_1');
+  //  let data = JSON.parse(article.dataset.plantedPlants)
+  console.log(article.dataset.plantedplants);
+  const draw = SVG().addTo('#circle_1').size(900, 300);
+  console.log(shape);
+  if (shape == 'Rtriangle') {
+    var triangle = draw.polygon('0,0 900,1300 900,0').fill(moduleColor).stroke({
+      width: 1
+    });
+  }
+  if (shape == 'Ltriangle') {
+    var triangle = draw.polygon('0,0 900,300 900,0').fill(moduleColor).stroke({
+      width: 1
+    });
+  }
+  if (shape == 'rectangle') {
+    const rect = draw.rect(900, 300).attr({
```

Comment:
there's actually no reason to assign this to a variable, at all, right? `rect` isn't accessible outside this `if` block